app/pb_kits/playbook/pb_project/_project.jsx

```diff
@@ -0,0 +1,96 @@
+/* @flow */
+
+import React from 'react'
+import classnames from 'classnames'
+import DateTime from '../pb_kit/dateTime.js'
+import { Body, Caption, Icon, Title } from '../'
+
+const dateString = (value: DateTime) => {
+  const month = value.toMonthNumber()
+  const day = value.toDay()
+
+  return ` · ${month}/${day}`
+}
+
+type ProjectProps = {
+  dark?: Boolean,
+  date: String,
+  link?: String,
+  projectName: String,
+  projectNumber: Number,
+}
```

Comment:
@jasoncypret @thestephenmarshall, I am wondering something, should I add to type ProjectProps 
  ```
  className?: String,
  data?: String,
  id?: String,
```